src/features/groups/components/GroupCardList/GroupCard.tsx

```diff
@@ -0,0 +1,27 @@
+import { Card } from '@/components/ui';
+import { fetchContainerList } from '@/lib/api/data';
+import { fetchUserList } from '@/lib/api/data';
+import { IContainer, IGroup, IUser } from '@/types/definition';
+
+import Link from 'next/link';
+
+import { ContainerCount } from './ContainerCount';
+import { UserCount } from './UserCount';
+
+export const GroupCard = async ({ group }: { group: IGroup }) => {
```

Comment:
It may be better to pass only the necessary props individually.

If you pass the entire `IGroup` to this component, this component might receive an unnecessary prop when a new property is added to `IGroup` in the future.

Please keep in mind the ISP (Interface Segregation Principle) of the SOLID principles:
- [JP article](https://zenn.dev/koki_tech/articles/361bb8f2278764#%E2%9C%85-isp-(interface-segregation-principle))
- [EN article](https://betterprogramming.pub/how-to-apply-interface-segregation-principle-in-reactjs-fadf77113c5d)